docs/tutorials/join-dal-baker/verify-rights.md

```diff
@@ -46,26 +55,39 @@ Follow these steps to verify that your DAL node is receiving attestation rights:
    Ensure that you are registered as a delegate and stake more tez, retaining a small amount for transaction fees.
    If necessary you can get more from the faucet.
 
-   - Check to see if you will receive rights two cycles in the future:
+   - Check to see if you will receive rights two cycles in the future, using commands similar to those above for the current cycle.
+   You can see who will receive rights no farther than two cycles in the future.
+   This number of cycles is set by the `consensus_rights_delay` network parameter.
 
-      1. Run this command to get the current cycle:
+     If this returns a list of future attestation rights for your account, the delay has not expired yet and you must wait for that cycle to arrive.
 
-         ```bash
-         octez-client rpc get /chains/main/blocks/head | jq | grep '"cycle"'
-         ```
+     You can find when the next cycle will start by running these commands:
 
-      1. Add two to the cycle and run this command to see what rights your account will have in that cycle.
-      For example, if the current cycle is 149, run this command to get its rights in cycle 151:
+        1. Find the last level of the current cycle by running this command:
 
-         ```bash
-         octez-client rpc get /chains/main/blocks/head/helpers/baking_rights\?cycle=151\&delegate=$MY_BAKER\&max_round=2
-         ```
+           ```bash
+           octez-client rpc get "/chains/main/blocks/head/helpers/levels_in_current_cycle"
+           ```
+
+         1. Pass the last level of the cycle as the `<last-block>` parameter in this command:
 
-         If this command returns a list of future attestation rights for your account, the delay has not expired yet and you must wait for that cycle to arrive.
+            ```bash
+            octez-client rpc get "/chains/main/blocks/head/helpers/attestation_rights?level=<last-block>" | grep '"estimated_time"'
```

Comment:
We should also point at attestations rights for DAL:

```
/chains/main/blocks/head/context/dal/shards
```

To get the DAL committee for the head.

Reply:
This item is only about when the next cycle starts, not about any rights. We use the attestation_rights RPC just to get a timestamp, not for the rights themselves.